packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -674,7 +676,6 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
     if (!mapEquals<ShortcutActivator, Intent>(_shortcuts, value)) {
       _shortcuts = value;
       _indexedShortcutsCache = null;
-      notifyListeners();
```

Comment:
I think we want to keep this here to keep the (newly) documented contract? Otherwise, the ShortcutManager would never notify any listeners?